.circleci/config.yml

```diff
@@ -21,13 +24,28 @@ jobs:
           name: Run ESLint
           command:
             npm run lint
+      
+      - run:
+          name: Unit tests
+          command:
+            npm run test:unit
 
       - run:
-          name: Build
+          name: Component tests
           command:
-            npm run build
+            npm run test:comp
+
+      - run:
+          name: E2E tests
+          command: |
+            DocID=$(curl -s --data-binary @data/drugcases.qvf http://localhost:9076/v1/apps/import | jq -r '.attributes.id')
+            npx parcel src/index.html &
+            DocID=$DocID npm run test:e2e
 
       - save_cache:
           paths:
             - node_modules
-          key: npm-dependency-cache-{{ checksum "package.json" }}
+          key: npm-dependency-cache-{{ checksum "package-lock.json" }}
+
+      - store_artifacts:
+          path: test/e2e/__artifacts__
```

Comment:
Why store these, aren't they supposed to be in the git repo? Or are these for CCI artefacts when something fails?

Reply:
Exactly - they will be possible to view in the circleCI UI. So you don't need to ssh into the build container to view the images (baseline/diff/regression)